app/models/packages_location.js

```diff
@@ -0,0 +1,19 @@
+import Ember from "ember";
+import Model from 'ember-data/model';
+import attr from 'ember-data/attr';
+import { belongsTo } from 'ember-data/relationships';
+
+export default Model.extend({
+  package_id: attr('number'),
```

Comment:
CamelCase is preferred in Ember as Ember maps snake_case JSON data to CamelCase...